src/lib/pages/frontsite/DocumentRequests/DocumentRequestForm.js

```diff
@@ -67,6 +67,7 @@ class DocumentRequestForm extends Component {
       paymentInfo,
       notes,
       publisher,
+      location: { state },
```

Comment:
I'm not sure how to understand the usage of this prop, how is it a state of a location ?

Reply:
when navigating to a location react allows you to pass props which end up being stored in the location's state property, i guess it often does contain object that change the state of the location/page